### PR TITLE
Protractor tests suites

### LINKED ISSUE #19: Setup protractor tests (as close as possible to React)

### PR DESCRIPTION
**Pull request by sahbi-ktifa**
Protractor tests suites for account/administration.
Review generator architecture to stick to master.

Will fix #19

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
I think I know why your PR failed here.
It's because our Travis builds use `maven` and when generating a Vue.js project, and launching `npm run webpack:build`, all files are generated into `dist/` instead of `target/www`.

Good news, the PR by William will resolve this problem.
But it won't work anymore for gradle as `target/www` is hard coded -> I will do another PR to update this.

**Comment by sahbi-ktifa** [comment]
Yes I saw that, that's why I modify build to deploy on target/www already, but no luck so far. I'll check this out tomorrow 😉

File: generators/client/templates/vue/config/index.js [hunk 1]
```diff
@@ -38,10 +38,10 @@ module.exports = {
 
     build: {
     // Template for index.html
-        index: path.resolve(__dirname, '../dist/index.html'),
+        index: path.resolve(__dirname, '../target/www/index.html'),
 
         // Paths
-        assetsRoot: path.resolve(__dirname, '../dist'),
+        assetsRoot: path.resolve(__dirname, '../target'),
```
[the remark this answers is not in the document]
Comment (sahbi-ktifa) [reply generators/client/templates/vue/config/index.js]: Hum...  I should definitely try this! 😁

**Comment by sahbi-ktifa** [comment]
Hey guys, this is a lot of works but this helped me reviewing the whole app. As I made lots of refactoring / modifications, please consider reviewing this asap as this will impact all current work in progress.

Thanks !

**Comment by pascalgrimaud** [comment]
@sahbi-ktifa : I think it will be hard to do a review in detail. I would suggest to merge it, if you're confident, and we can improve it later. As Vue.js is still in WIP, I don't care if the build is broken

**Comment by sahbi-ktifa** [comment]
Builds are all green now :)
Ok then, I merge it.

**Comment by pascalgrimaud** [comment]
Oh you add service, nice !
I will rework my elasticsearch branch to include this major change and will PR later

**Pull request merged**